libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/TelemetryEventUserFeedback.kt

```diff
@@ -0,0 +1,60 @@
+package com.mapbox.navigation.core.telemetry
+
+import android.location.Location
+import com.google.gson.Gson
+
+internal enum class FeedbackType(val feedbackType: String) {
+    FEEDBACK_TYPE_ACCIDENT("accident"),
```

Comment:
don't see `general` event, shouldn't it be here?

Reply:
👍

Reply:
👍